Library/Homebrew/cmd/audit.rb

```diff
@@ -572,6 +573,18 @@ def audit_specs
     end
   end
 
+  def audit_revision
+    return unless formula.tap # skip formula not from core or any taps
+
+    fv = FormulaVersions.new(formula)
+    revision_map = fv.revision_map("origin/master")
```

Comment:
That probably seems far-fetched, but I think this would fail if someone installed Homebrew from a tarball (i.e. without using `git`) then ran `brew audit <some formula>`.